RSDKv5/RSDK/Graphics/GLFW/GLFWRenderDevice.cpp

```diff
@@ -90,6 +90,10 @@ bool RenderDevice::Init()
         PrintLog(PRINT_NORMAL, "ERROR: [GLFW] window creation failed");
         return false;
     }
+    if (videoSettings.windowed) {
+        const GLFWvidmode *mode = glfwGetVideoMode(glfwGetPrimaryMonitor());
+        glfwSetWindowPos(window, (mode->width - videoSettings.windowWidth) / 2, (mode->height - videoSettings.windowHeight) / 2);
+    }
```

Comment:
On my setup (dual 1080p monitors, primary monitor on the right, X11/KDE Plasma), `glfwGetPrimaryMonitor()` returns my primary monitor but `glfwSetWindowPos()` moves the window to my secondary monitor (left), supposedly because `glfwSetWindowPos()` uses global screen coordinates.
Moreover, it appears that this function [doesn't work on Wayland](https://www.glfw.org/docs/latest/group__window.html#ga1abb6d690e8c88e0c8cd1751356dbca8).

Reply:
I don't have a multi-monitor setup to test this with, but I may have fixed it. Try it now.

In Wayland, window position is completely controlled by the compositor with absolutely no way for clients to request a position (baffling design choice, I know), so `glfwSetWindowPos` has no effect and the window will not be centered. Everything still works as before.